### PR TITLE
Add posts app and models

### PR DESCRIPTION
**Pull request by arthurarty**

### CODE REVIEW

File: apps/posts/models.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    name = models.CharField(max_length=60, unique=True)
+    description = models.TextField(null=True, blank=True)
+
+
+class Posts(models.Model):
+    title = models.CharField(
+        max_length=255,
+        error_messages={"required": "Title is required when creating post."}
+    )
+    body = models.TextField(
```
Comment (skapeyi) [inline apps/posts/models.py:15]: add `excerpt` which is a summary of the post

File: apps/posts/models.py [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+        error_messages={"required": "Add a body for your article."}
+    )
+    image_url = models.URLField(null=True, blank=True)
```
Comment (skapeyi) [inline apps/posts/models.py:18]: Change this to `feature_img_url`.

File: apps/posts/models.py [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+    slug = models.SlugField(max_length=255, unique=True)
+    author = models.ForeignKey(
+        User, on_delete=models.CASCADE, related_name="posts"
+    )
+    category = models.ForeignKey(
+        Category, on_delete=models.CASCADE, related_name="posts"
```
Comment (skapeyi) [inline apps/posts/models.py:24]: A post can have many categories. Change this to a one to many

**Review by skapeyi — CHANGES_REQUESTED** [review]

**Comment by arthurarty** [comment]
@skapeyi Hey done implementing those changes.

**Comment by skapeyi** [comment]
@arthurarty I don't see the updates on the PR.

**Comment by arthurarty** [comment]
> @arthurarty I don't see the updates on the PR.

@skapeyi Hey I had forgotten to push. I have pushed now.

**Review by skapeyi — APPROVED** [review]

**Pull request merged**